tck/src/test/java/io/smallrye/opentelemetry/tck/arquillian/SmallRyeBeanArchiveHandler.java

```diff
@@ -0,0 +1,59 @@
+package io.smallrye.opentelemetry.tck.arquillian;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipInputStream;
+
+import javax.annotation.Priority;
+
+import org.jboss.weld.environment.deployment.discovery.BeanArchiveBuilder;
+import org.jboss.weld.environment.deployment.discovery.BeanArchiveHandler;
+import org.jboss.weld.environment.util.Files;
+
+/**
+ * @author Pavol Loffay
+ */
+@Priority(1)
+public class SmallRyeBeanArchiveHandler implements BeanArchiveHandler {
+
+    @Override
+    public BeanArchiveBuilder handle(String beanArchiveReference) {
+
+        // An external form of a wildfly beans.xml URL will be something like:
```

Comment:
Is this only needed because it's being tested on WF?

Reply:
This class can be completely removed. The TCK is passing fine without it.